roles/filetree_create/templates/current_workflow_job_templates.j2

```diff
@@ -25,7 +25,7 @@ controller_workflows:
         all_parents_must_converge: "{{ node.all_parents_must_converge }}"
 {% if node.extra_data is defined and node.extra_data | length > 0 %}
         extra_data:
-          {{ node.extra_data | to_nice_yaml(sort_keys=false) | indent(10) | replace("'{{", "!unsafe \'{{") | replace("$encrypted$", "\'\'") }}
```

Comment:
`replace("$encrypted", "\'\'")` is needed to solve issue #8... isn't it needed anymore?

Reply:
Yes, I was resolving conflicts due to missing the newest changes and didn't notice, thanks.